src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -313,5 +281,59 @@ internal static void UpdateRequestBestMatchingToolPathsIfAvailable(RestoreReques
                 }
             }
         }
+
+        internal static List<string> GetRestoreOutput(RestoreRequest request, LockFile lockFile)
+        {
+            IEnumerable<string> GetFiles(string packageId, NuGetVersion version, IEnumerable<VersionFolderPathResolver> resolvers)
```

Comment:
This doesn't need to be a local function does it? 
Makes it harder to read tbh, as the method appears longer thna it actually is.

Reply:
It just made it so I'd have to pass around less objects, I'll make it its own method.